src/pseudo_intrinsics.h

```diff
@@ -154,8 +151,11 @@ static inline __mmask64 _mm512_cmpeq_epi8_mask(__m512i a, __m512i b)
 	for (i = 0; i < 64; ++i)
 		mask |= ((ma[i] == mb[i]) << i);
 
-	return (__mmask64)mask;
+	return mask;
 }
+#ifndef _mm512_cmpeq_epi8_mask // needed by gcc-4.9
+#define _mm512_cmpeq_epi8_mask(a, b) _mm512_cmp_epi8_mask(a, b, _MM_CMPINT_EQ)
```

Comment:
`_mm512_cmpeq_epi8_mask` is defined as a macro in icc's header. Defining a function with the same name would cause compile error.